packages/strapi-provider-email-sendmail/README.md

```diff
@@ -51,3 +52,36 @@ module.exports = ({ env }) => ({
   // ...
 });
 ```
+
+### Example with DKIM
+
+Using **DKIM** (DomainKeys Identified Mail) can prevent  emails from being considered as spam. More details about this subject  can be found in the discussion on the Strapi forum:  [Unsolved problem: emails goes to spam!](https://forum.strapi.io/t/unsolved-problem-emails-goes-to-spam/512?u=soringfs)
+
+#### Generate the keys using OpenSSL
+
+```perl
+openssl genrsa -out dkim-private.pem 1024
+openssl rsa -in dkim-private.pem -pubout -out dkim-public.pem
+```
+
+**Path -** `config/plugins.js`
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  email: {
+    provider: 'sendmail',
+    providerOptions: {
+      dkim: {
+        privateKey: 'replace-with-dkim-private-key',
+        keySelector: 'abcd', // the same as the one set in DNS txt record, use online dns lookup tools to be sure that is retreivable
```

Comment:
To merge a PR like this, these steps need to be documented somewhere. @pwizla how should we handle this as it falls in the topic of moving provider documentation to the actual documentation?

Reply:
I totally missed this notification, Derrick, sorry!

I need to think about it and probably discuss it with Mégane & Jim. 
Adding this to my todolist, thanks for your patience.